### ISSUE TITLE
Blender-Importer error with some boards: edges.new(): this edge exists

### ISSUE THREAD

**Issue by Haschtl** [30350n/pcb2blender#31]
I encountered an issue with the `improve_board_mesh` function: `edges.new(): this edge exists` in the statement `bm.edges.new((top_vert, bot_vert))`.

I think, this error can occur for boards with complex outlines, that may be not defined properly.

In my case it works with a simple try/except:
```python
            try:
                bm.edges.new((top_vert, bot_vert))
            except ValueError as e:
                print(e)
```

**Comment by 30350n** [comment]
That's a bad solution imo, especially because I already got a check which *should* catch this right above [here](https://github.com/30350n/pcb2blender/blob/master/pcb2blender_importer/importer.py#L744). So this shouldn't really happen. Though iirc I'm kind of abusing the way KiCad writes the PCB mesh with that code, so if they ever change something about that it might not work anymore. Can you share the pcb3d file for this one? 😅

**Comment by Haschtl** [comment]
Yes, not the best solution... how about this:
```
            existing=bm.edges.get((top_vert, bot_vert))
            if existing is None:
                bm.edges.new((top_vert, bot_vert))
```

Unfortunately, I cannot share this pcb3d file :(

**Comment by 30350n** [comment]
Can you share it privately? (There's a mailto link at the bottom of [my website](https://30350n.de/)). Also, the `pcb.wrl` file from the .pcb3d file would be enough probably (.pcb3d files are just .zip files so you should be able to extract it quite easily).

**Comment by 30350n** [comment]
Should be fixed in [v2.7](https://github.com/30350n/pcb2blender/releases/tag/v2.7-k7.0-b3.6) now!

**Comment by 30350n** [comment]
Had any time to test this again? @Haschtl
